### PR TITLE
Update CV link

### PR DESCRIPTION
**Pull request by 62442katieb**
I accidentally broke my CV link. 🙃 Should be good-to-go now.

### CODE REVIEW

File: posters/_posts/2019-06-12-bottenhorn-physics-poster.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: poster
+title: "Large-scale brain networks underlying domain-specific
+memory, intelligence, and academic performance"
+nickname: 2019-06-12-bottenhorn-physics-poster
+authors: "Bottenhorn KL, Salo T, Sutherland MT, Laird AR"
+year: "2019"
+conference: "HBM"
+image: /assets/images/posters/2019-06-12-bottenhorn-physics-poster.png
```
Comment (tsalo) [inline posters/_posts/2019-06-12-bottenhorn-physics-poster.md]: This image doesn't exist.

**Review by tsalo — CHANGES_REQUESTED** [review]
Not sure if the figure already exists under a different name or if you need to add it.

**Comment by tsalo** [comment]
The poster page you're adding seems to be a mashup of ["Quantitative comparison of functional decoding approaches across meta-analytic frameworks"](https://nbclab.github.io/posters/bottenhorn-decoding-poster) and ["Large-scale brain networks underlying domain-specific memory, intelligence, and academic performance"](https://nbclab.github.io/posters/bottenhorn-networks-ohbm). The title and image seem to be for the latter, while the author list, fulltext, and abstract match the former.

**Comment by 62442katieb** [comment]
I'm on it. Looks like I just hadn't finished updating the poster page and it was abandoned. 😢 Will fix later today

**Comment by 62442katieb** [comment]
Turns out it was actually a weird duplicate poster page. Fixed! (And GitHub link updated!)

**Pull request merged**